### ISSUE TITLE
atricore.com is no longer available

### ISSUE THREAD

**Issue by timcharper** [atricore/iamtf-examples#1]
Trying to run terraform to get the examples running, and I get this:

```
 terraform init

Initializing the backend...

Initializing provider plugins...
- Finding atricore.com/iam/iamtf versions matching "~> 0.5.0"...
╷
│ Error: Failed to query available provider packages
│ 
│ Could not retrieve the list of available versions for provider atricore.com/iam/iamtf: could not connect to atricore.com: Failed to request discovery document: Get
│ "https://atricore.com/.well-known/terraform.json": x509: certificate is not valid for any names, but wanted to match atricore.com
╵
```

**Comment by sgonzalezoyuela** [comment]
Hi Tim,

You can manually install the provider in your environment as a work-around.  Basically you extract the plugin to the TF plugins folder that matches your architecture:

```sh
unzip ~/Downloads/terraform-provider-iamtf-linux-amd64-0.5.0.zip
mkdir -p ~/.terraform.d/plugins/atricore.com/iam/iamtf/0.5.0/linux_amd64
cp terraform-provider-iamtf ~/.terraform.d/plugins/atricore.com/iam/iamtf/0.5.0/linux_amd64
```
More details here:
https://iam.tf/guide/quick-start.html#install-terraform-plugin

We are working on publishing the plugin so it can be automatically downloaded by Terraform.

Let me know if this works for you.

Cheers!

**Comment by sgonzalezoyuela** [comment]
We released version 0.6.x of the TF plugin.  Now you can use `terraform init` command to automatically get the plugin from TF repository.  You must change the plugin source and version to be `atricore/iamtf` and 0.6.0 or greater.
